md/runtime-monitoring.md

```diff
@@ -0,0 +1,102 @@
+You will see, in the `# Monitoring` section, two series of properties with their default value:
+
+    ## Monitoring
+    ## publish metrics to JMX:
+    #org.bonitasoft.engine.monitoring.jmx.enable=true
+    ## periodically print metrics to logs (bonita-related metrics only):
+    #org.bonitasoft.engine.monitoring.logging.enable=false
+    ## print to logs every minute by default (in the ISO-8601 duration format):
+    #org.bonitasoft.engine.monitoring.logging.step=PT1M
+    #
+    ## publish metrics related to JVM memory:
+    #org.bonitasoft.engine.monitoring.metrics.jvm.memory.enable=false
+    ## publish metrics related to JVM Threads:
+    #org.bonitasoft.engine.monitoring.metrics.jvm.threads.enable=false
+    ## publish metrics related to JVM garbage collection:
+    #org.bonitasoft.engine.monitoring.metrics.jvm.gc.enable=false
+    ## publish technical metrics related to Worker / Connector thread pools:
+    #org.bonitasoft.engine.monitoring.metrics.executors.enable=false
+
+The first series is the list of **publishers** that can be activated.  
+In Community edition, 2 publishers are provided:
+* JMX (enabled by default), that allows to use any JMX console to monitor your favorite metrics (except JVM metrics)
+* Logging (disabled by default), that prints to standard Bonita log file the Bonita-related metrics. Print interval can
+be changed (property `org.bonitasoft.engine.monitoring.logging.step`).
+
+The second series is the list of **metrics** (counters) that can be exposed.  
+All configurable metrics are disabled by default and can be enabled separately.  
+They provide information about:
+* the running JVM memory
+* JVM threads
+* Garbage collection usage
+* Worker / Connector thread pools
+
+Each of these metrics provides many different counters to finely understand what is going on.
+
+To change any value, **uncomment the line by removing the # character**, and change the true / false value.  
+Server restart is required for the changes to take effect.
+
+## Subscription-only monitoring publisher
+
+Additionally, Bonita Subscription editions can publish to a REST endpoint in the
+[Prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-example), that can
+easily be consumed by tools like Grafana, etc.
+
+To activate, simply edit file `./setup/platform_conf/current/platform_engine/bonita-platform-sp-custom.properties`
+and change:
+  
+    # Activate publication of metrics to prometheus:
+    # com.bonitasoft.engine.plugin.monitoring.prometheus.enable=false
+
+to
+
+    # Activate publication of metrics to prometheus:
+    com.bonitasoft.engine.plugin.monitoring.prometheus.enable=true
+
+and restart server.
+
+This exposes all activated metrics (see [above](#activating-specific-monitoring-metrics)) at endpoint:
+
+    http://`<SERVER_URL>`/bonita/metrics  
+
+Use this URL to configure your graphical monitoring tool in order to record and display the metrics.
```

Comment:
You might want to give some clues to our beloved users here (or be asked to).
For your information we had to write a whole Groovy tutorial in the documentation to help user define variables and stuff in groovy cause it's not easy for them to find the right documentation, so I guess it's the same here.

Reply:
you mean how to configure Grafana?

Reply:
At least a link to the prometheus documentation since we put forward this solution

https://prometheus.io/docs/prometheus/latest/getting_started/
https://prometheus.io/docs/prometheus/latest/installation/

Maybe there is no need I don't know, it depends on which kind of users we target for this feature I guess..

Reply:
WIll be nice to have a how to plug prometheus and grafana, but it is out of this first value

Reply:
Created https://bonitasoft.atlassian.net/browse/BR-226 for that.